templates/patterns/banners/pattern-banner.html.twig

```diff
@@ -0,0 +1,50 @@
+{#
```

Comment:
All of the 4 variant files are the same, there is too much code duplication. You can just check the variant (its a variable that is available in the templates) and pass the image information if the variant is image or image_shade.